### ISSUE TITLE
Add truncation to the Ebola vignette

### ISSUE THREAD

**Issue by seabbs** [epinowcast/epidist#485]
The Ebola vignette currently has no features that really look at truncation and also doesn't mention this. We could do one of two things: 

1. Keep it as a retrospective study, talk about this clearly and focus on making another doc that looks at truncation in a real world setting
2. Update it to use truncated data, this probably means filtering at a point in time, comparing between the truncated and untruncated data etc.

**Comment by athowes** [comment]
I don't think it would be a big lift to do 2. here. This was something I had been intending to do, but scrapped it to get it out the door.

**Comment by seabbs** [comment]
Done in #513
